ceph.spec.in

```diff
@@ -238,6 +238,7 @@ BuildRequires:	xfsprogs-devel
 BuildRequires:	xmlstarlet
 BuildRequires:	nasm
 BuildRequires:	lua-devel
+BuildRequires:  yaml-cpp-devel >= 0.6
```

Comment:
personally, i'd put
```
%if 0%{with seastar} || 0%{with jaeger}
BuildRequires: yaml-cpp-devel >= 0.6
$endif
```
and could you please s/space/tab/ for more consistency with the other `BuildRequires:` lines?

Reply:
also, could you please fix the syntax error and typo in the commit message?